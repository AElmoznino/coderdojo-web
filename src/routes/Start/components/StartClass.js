import React from 'react'
import styled from 'styled-components'

const Wrap = styled.div`
  background-color: ${({ theme }) => theme.colors.white};
  min-height: 100px;
`

export const StartClass = () => (
  <Wrap>
    <p>Lektion 1</p>
    <img src="https://placeimg.com/200/200/tech" />
    <p>Här får du lära dig mer om ...</p>
  </Wrap>
)

export default StartClass
