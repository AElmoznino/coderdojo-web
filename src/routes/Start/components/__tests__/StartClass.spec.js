import React from 'react'
import { shallow } from 'enzyme'
import StartClass from '../StartClass'

describe('components/StartClass', () => {
  let component

  beforeEach(() => {
    component = shallow(<StartClass />)
  })

  it('renders StartClass', () => {
    expect(component).toMatchSnapshot()
  })
})
