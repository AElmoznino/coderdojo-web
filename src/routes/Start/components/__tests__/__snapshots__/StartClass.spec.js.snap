// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/StartClass renders StartClass 1`] = `
<styled.div>
  <p>
    Lektion 1
  </p>
  <img
    src="https://placeimg.com/200/200/tech"
  />
  <p>
    Här får du lära dig mer om ...
  </p>
</styled.div>
`;
